hydradx/tests/test_omnipool_amm.py

```diff
@@ -1251,52 +1283,56 @@ def test_volatility_limit(omnipool: oamm.OmnipoolState):
 def test_LP_limits(omnipool: oamm.OmnipoolState, max_withdrawal_per_block, max_lp_per_block):
     omnipool.max_withdrawal_per_block = max_withdrawal_per_block
     omnipool.max_lp_per_block = max_lp_per_block
-    agent = Agent(holdings={'HDX': 10000000000})
+    state = omnipool.copy()
```

Comment:
What is the point of this copy, since the original is never used again? Is it just for debugging purposes?

Reply:
You're right, I see that now.